src/pods/canvas/canvas.business.ts

```diff
@@ -73,3 +79,75 @@ export const calculateRelationXCoordinate = (
   xOrigin: calculateRelationXCoordinateOrigin(tableOrigin, tableDestination),
   xDestination: calculateRelationXCoordinateEnd(tableOrigin, tableDestination),
 });
+
+const seekField = (
```

Comment:
Since `SeekResult` is only going to be used inside this function, we can define the interface just before the `SeekField`declaration instead of the _vm_, why this approach? We place in the _vms_ entities that we expect that are going to be used in several files, in this case is rare that this interface could be used outside this file.

```diff
+ interface seekResult {
+  found: boolean;
+ parentCollapsed: boolean;
+  YPosition: number;
+ }

const seekField = (
```

> Note you can export in case is needed

Reply:
It's fixed